lib/reporting/cloudwatch_client.rb

```diff
@@ -141,12 +151,13 @@ def show_progress?
 
     private
 
-    def log(message)
+    def log(level, message)
       if logger
-        if @progress_bar
-          @progress_bar.log(message)
+        int_level = Logger.const_get(level.upcase)
+        if @progress_bar && int_level >= logger.level
+          @progress_bar.log("#{level.upcase}: #{message}")
```

Comment:
Unfortunately, the Progress Bar logging method does not have a notion of "levels" so we have to kind of recreate some of the semantics of Logger class here (and the gem is not likely to change soon) so this was my compromise (see https://github.com/jfelchner/ruby-progressbar/issues/169)

Context: I wanted the warnings to show up when running this rake task along with the progress bar, but not the rest of the info/debug-type log messages